op_crates/console/02_console.js

```diff
@@ -907,22 +907,11 @@
     inspectOptions,
   ) {
     if (customInspect in value && typeof value[customInspect] === "function") {
-      return String(value[customInspect]());
-    }
-    // This non-unique symbol is used to support op_crates, ie.
-    // in op_crates/web we don't want to depend on unique "Deno.customInspect"
-    // symbol defined in the public API. Internal only, shouldn't be used
-    // by users.
-    const nonUniqueCustomInspect = Symbol.for("Deno.customInspect");
-    if (
-      nonUniqueCustomInspect in value &&
-      typeof value[nonUniqueCustomInspect] === "function"
-    ) {
```

Comment:
Removing this logic means that `Deno.customInspect` will stop working. I think we should still support it until 2.0

Reply:
The non-unique custom inspect which we use for op crates implementations should be renamed to something else since the public custom inspect symbol has stolen its name. We handle the internal one with its own logic (i.e. pass it the `inspect()` function as an arg) so the difference should be preserved.

Reply:
I think we should always pass the `inspect` function as an arg - even for the public one.

Reply:
OK. I'll rename internal one to something else and keep the different handlings of public one and internal one.